app/navigation/Navigator.scala

```diff
@@ -16,29 +16,43 @@
 
 package navigation
 
-import javax.inject.{Inject, Singleton}
-import play.api.mvc.Call
+import config.Constants
+import config.Constants.MaxSchemeBankAccounts
+import config.Refined.OneToTen
 import controllers.routes
-import pages.{CheckReturnDatesPage, _}
+import eu.timepit.refined._
+import eu.timepit.refined.api.Refined
 import models._
+import pages._
+import play.api.mvc.Call
+
+import javax.inject.{Inject, Singleton}
 
 @Singleton
 class Navigator @Inject()() {
 
-
-
   private val normalRoutes: Page => UserAnswers => Call = {
     case StartPage(srn)             => _ => routes.SchemeDetailsController.onPageLoad(srn)
     case SchemeDetailsPage(srn)     => _ => routes.CheckReturnDatesController.onPageLoad(srn, NormalMode)
 
     case page@CheckReturnDatesPage(srn)  => ua =>
       if(ua.get(page).contains(true))
-        routes.SchemeBankAccountController.onPageLoad(srn, NormalMode)
+        routes.SchemeBankAccountController.onPageLoad(srn, refineMV[OneToTen](1), NormalMode)
       else
         routes.UnauthorisedController.onPageLoad
-
-    case SchemeBankAccountPage(srn) => _ => routes.UnauthorisedController.onPageLoad
-    case _                          => _ => routes.IndexController.onPageLoad
+    case SchemeBankAccountPage(srn) => _ => routes.SchemeBankAccountSummaryController.onPageLoad(srn)
+    case SchemeBankAccountSummaryPage(srn, addBankAccount @ true) => userAnswers =>
```

Comment:
`addBankAccount @` isn't needed

Reply:
this was more for readability buy can remove